packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -146,6 +148,22 @@ export interface RecordingCallFeatureState {
   isRecordingActive: boolean;
 }
 
+/* @conditional-compile-remove(raise-hands) */
+/**
+ * State only version of {@link @azure/communication-calling#RaiseHandCallFeature}. {@link StatefulCallClient} will
+ * automatically listen for raised hands on the call and update the state exposed by {@link StatefulCallClient} accordingly.
+ *
+ * @public
+ */
+export interface RaiseHandCallFeatureState {
+  /**
+   * Proxy of {@link @azure/communication-calling#RaiseHandCallFeature.raisedHands}.
+   */
+  allRaisedHands: RaisedHand[];
```

Comment:
should we just call the property raisedHands instead of allRaisedHands?

raisedHands: Raisedhand[]